main.js

```diff
@@ -41,20 +46,31 @@ var mantras = [
 
 receiveMessage.addEventListener('click', getMessage);
 
+inputName.addEventListener('click', showPage);
 
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 };
 
 function getMessage(event) {
   event.preventDefault();
+
```

Comment:
Consider looking into different ways the getRandomIndex() function can be written to potentially clean up the readability -

affirmations[getRandomIndex(affirmations)] - might be a little confusing when used in denser functions